data.py

```diff
@@ -960,6 +961,26 @@ def get_se_access_token(self):
         logging.debug(f"SE ACCESS TOKEN: {se_access_token}")
         return se_access_token, base_secure_endpoint_url
 
+    def recommend_exclusions(self):
+        '''        
+        Loop through the processes seen on the endpoint and compare them with Cisco Maintained Exclusions.
+        If any matches are found, return them.
+        TODO - Once an API call is implemented to pull details for the Cisco Maintained Exclusions, use that call instead of this hard coded json 
+        '''
+
+        recommendations = {"Microsoft Windows Default": ["Always Recommended"]}
+        processes = self.get_top_processes()
+        if processes != "":
+            for process_line in processes.split("\n"):
+                process = process_line.split("|")[1].split("\\")[-1].lower()
+                for program in TEMP_MAINTAINED_EXCLUSIONS:
+                    if process in TEMP_MAINTAINED_EXCLUSIONS[program]:
+                        if program in recommendations:
+                            recommendations[program].append(process)
+                        else:
+                            recommendations[program] = [process]
+        return recommendations

```

Comment:
recommend running psutil directly instead of self.get_top_processes()

replace lines 971-982 with the following, should work

```
        recommendations = {"Microsoft Windows Default": ["Always Recommended"]}

        for proc in psutil.process_iter():
            try:
                process = proc.name()
                for program in TEMP_MAINTAINED_EXCLUSIONS:
                    if process in TEMP_MAINTAINED_EXCLUSIONS[program]:
                        if program in recommendations:
                            recommendations[program].append(process)
                        else:
                            recommendations[program] = [process]
            except psutil.NoSuchProcess:
                logging.warning("Error: psutil.NoSuchProcess")

        return recommendations
```